### PR TITLE
Allows Hydraulic Clamps to pick up any `/obj/structure` at Oranges request

### PR DESCRIPTION
**Pull request by SuperNovaa41**
image
image

:cl:
balance: You can now pickup any structure in a mech.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
that's the modsuit module is that what mechs use

File: code/modules/mod/modules/modules_supply.dm [hunk 1]
```diff
@@ -51,7 +51,7 @@
 		return
 	if(!mod.wearer.Adjacent(target))
 		return
-	if(istype(target, /obj/structure/closet/crate) || istype(target, /obj/item/delivery/big))
+	if (isstructure(target))
```
Comment (Iamgoofball) [inline code/modules/mod/modules/modules_supply.dm:54]: you did this in the modsuits 2021 by @Fikou hydraulic clamp, and not the ripley one

Comment (SuperNovaa41) [reply code/modules/mod/modules/modules_supply.dm:54]: whoops

**Comment by SuperNovaa41** [comment]
Oranges is a fool and you can already do this as long as its unanchored

**Comment by JohnFulpWillard** [comment]
the supermatter isn't anchored

**Comment by Addust** [comment]
the SUPERMATTER SHARD is unanchored

**Pull request closed without merging**